src/main/java/edu/kis/powp/jobs2d/drivers/adapter/OperateToCommand.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class OperateToCommand implements DriverCommand
+{
+    private int x;
+    private int y;
+    private Job2dDriver driver;
+
+    public OperateToCommand(Job2dDriver driver,int x_1, int y_2)
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).